src/chat/panes/chat-pane/template.pug

```diff
@@ -1,6 +1,6 @@
-shopping-card(ng-repeat = "card in $ctrl.owner.cards" card = "card" ng-if = "$ctrl.uptodateScreen && !card.isArchived && !$ctrl.archiveScreen && !$ctrl.setSearchScreen")
-shopping-card(ng-repeat = "card in $ctrl.owner.cards" card = "card" ng-if = "!$ctrl.uptodateScreen && card.isArchived && $ctrl.archiveScreen && !$ctrl.setSearchScreen")
-shopping-card(ng-repeat = "card in $ctrl.owner.searchResult" card = "card" ng-if = "$ctrl.setSearchScreen")
-div.buttonDelete(ng-click = "$ctrl.deleteAllCards()" ng-if="$ctrl.recycleBinScreen && !$ctrl.setSearchScreen && !$ctrl.userScreen") Hepsini Sil
-shopping-card(ng-repeat = "card in $ctrl.owner.deletedcards" card = "card" ng-if = "$ctrl.recycleBinScreen && !$ctrl.setSearchScreen && !$ctrl.userScreen")
-user-box(ng-repeat = "user in $ctrl.owner.users" user = "user" ng-if = "$ctrl.userScreen")
+shopping-card(ng-repeat = "card in $ctrl.owner.cards" card = "card" ng-if = "$ctrl.screenType == 1")
```

Comment:
`        this.EventType = {
            SET_ACTIVE_THREAD: 'set active thread',
            UPDATE: 'update',
            SET_ACTIVE_CHAT_BOX: 'set active chat box',
            NEW_MESSAGE: 'new message',
            ADD_CHAT_BOX: 'add chat box',
            REMOVE_CHAT_BOX: 'remove chat box'
        };
`
@ezgi, I have copied these codes from week-3-2 folder at chat-service.js. we may define enum like this in card-service.